src/igvfd/types/sample.py

```diff
@@ -321,6 +351,28 @@ def summary(self, request, sample_terms, donors, sex, age, age_units=None, embry
 
         return summary_terms.strip(',')
 
+    @calculated_property(schema={
+        'title': 'Parent Of',
+        'type': 'array',
+        'items': {
+            'title': 'Child Sample',
+            'type': ['string', 'object'],
+            'linkFrom': 'Sample.part_of',
+        },
+        'notSubmittable': True,
+    })
+    def parent_of(self, request, parent_of):
+        return paths_filtered_by_status(request, parent_of)
+
+    @calculated_property(schema={
+        'title': 'Pooled In',
+        'type': ['string', 'object'],
+        'linkFrom': 'Sample.pooled_from',
+        'notSubmittable': True,
+    })
+    def pooled_in(self, request, pooled_in):
+        return paths_filtered_by_status(request, pooled_in)
```

Comment:
This returns an array when I'd like it to just be a string since it will always just be a single link. Not sure how to accomplish this.

Reply:
Never mind, an array is fine since a sample may be pooled in multiple other samples.